### PR TITLE
Added SetTag operation to TagGroupEditor

### PR DESCRIPTION
**Pull request by linkmh**
Adding support for setTags for iOS and Android for Channel and Named User Tag Groups

### CODE REVIEW

File: js/TagGroupEditor.js [hunk 1]
```diff
@@ -40,6 +40,22 @@ class TagGroupEditor {
...
+  	* Overwrite the current set of tags on the Tag Group
+  	* @instance
+  	* @memberof TagGroupEditor
+  	* @function setTags
+  	*
+  	* @param {string} tagGroup The tag group.
+  	* @param {array<string>} tags Tags to set.
+  	* @return {TagGroupEditor} The tag group editor instance.
+  	*/
+  setTags(group: string, tags: Array<string>): TagGroupEditor {
```
Comment (rlepinski) [inline js/TagGroupEditor.js]: whitespace is weird

File: js/index.d.ts [hunk 2]
```diff
@@ -52,6 +52,17 @@ declare class TagGroupEditor {
...
+    /**
+  	* Overwrite the current set of tags on the Tag Group
+  	* @instance
+  	* @memberof TagGroupEditor
+  	* @function setTags
+  	*
+  	* @param {string} tagGroup The tag group.
+  	* @param {array<string>} tags Tags to set.
+  	* @return {TagGroupEditor} The tag group editor instance.
+  	*/
```
Comment (rlepinski) [inline js/index.d.ts]: fix whitespace

**Comment by rlepinski** [comment]
👍 After some whitespace cleanup

**Comment by marc-scig** [comment]
🐌

**Comment by crow** [comment]
🍔

**Pull request merged**